regression-test/suites/nereids_function_p0/agg_function/test_regr_sxx.groovy

```diff
@@ -0,0 +1,139 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_regr_sxx") {
+    sql """ DROP TABLE IF EXISTS test_regr_sxx_int """
+    sql """ DROP TABLE IF EXISTS test_regr_sxx_double """
+    sql """ DROP TABLE IF EXISTS test_regr_sxx_nullalble_col """
+
+
+    sql """ SET enable_nereids_planner=true """
+    sql """ SET enable_fallback_to_original_planner=false """
+
+    sql """
+        CREATE TABLE test_regr_sxx_int (
+          `id` int,
+          `x` int,
+          `y` int,
+        ) ENGINE=OLAP
+        Duplicate KEY (`id`)
+        DISTRIBUTED BY HASH(`id`) BUCKETS 4
+        PROPERTIES (
+        "replication_allocation" = "tag.location.default: 1"
+        );
+        """
+    sql """
+        CREATE TABLE test_regr_sxx_double (
+          `id` int,
+          `x` double,
+          `y` double,
+        ) ENGINE=OLAP
+        Duplicate KEY (`id`)
+        DISTRIBUTED BY HASH(`id`) BUCKETS 4
+        PROPERTIES (
+        "replication_allocation" = "tag.location.default: 1"
+        );
+        """
+    sql """
+        CREATE TABLE test_regr_sxx_nullalble_col (
```

Comment:
The table name `test_regr_sxx_nullalble_col` has a spelling mistake (`nullalble` should be `nullable`).

Reply:
get